docs/troubleshooting-development-issues.md

```diff
@@ -2,6 +2,8 @@ If you are facing issue, there is a high chance that the resolution is in this d
 
 ### Issues with installing the recommended prerequisites
 
+#### Disclaimer
+
```

Comment:
```suggestion

```

The whole project setup guide/documentation is a "disclaimer", that is there are no guarantees. We do not need legal-speak here.